lib/pbench/server/api/resources/users_api.py

```diff
@@ -114,13 +114,7 @@ def post(self):
             self.logger.info(
                 "New user registered, username: {}, email: {}", username, email_id
             )
-
-            response_object = {
-                "message": "Successfully registered.",
-            }
-            response = jsonify(response_object)
-            response.status_code = 201
-            return make_response(response, 201)
+            return "", 201
```

Comment:
I continually debate how completely we should try to follow REST conventions; most of the rules are fairly loose.

However, while I'd noticed before that you were using 201 here, for some reason this change just made me think about it more carefully. It's normally strongly recommended that a 201 return a `Location` response header giving the URI the client can use to retrieve the new resource ... in this case the user. Some recommend returning the resource as payload (_as if_ they'd done a `GET` already), or a human-readable HTML description.

Maybe not something we need to worry about right here, but maybe we should consider using `Location` somewhere along here for resources we create?

Reply:
Seems like a good thing to track in an issue to follow up in another PR?

Reply:
Opened https://github.com/distributed-system-analysis/pbench/issues/2145